test/rules/parameter_assignments.dart

```diff
@@ -45,3 +49,34 @@ void otherBad1(int parameter) { // LINT
   parameter += 3;
```

Comment:
Sorry, I guess I wasn't clear. I mean, didn't we decide to report lint violations at the point of the assignment rather than at the point at which the parameter is declared? In which case the lint should be produced for the reference to `parameter` on line 49 rather than at the declaration on line 48.